src/test/resources/validMigration.json

```diff
@@ -12,9 +12,14 @@
     ],
     "caseAttachments": [
       {
-        "path": "laborum nisi",
-        "label": "sunt",
-        "documentType": "deserunt laboris anim nisi"
+        "s3UntrustedUrl": "e7f5d229-3f23-450c-8f11-8ef647943ae3",
```

Comment:
Can this be renamed to something like `documentUuid`? `s3UntrustedUrl` alludes that it's something else.

We would never be uploading to the trusted directly do I don't see why we need untrusted here.

Reply:
Updated